routers/api/v1/repo/release_attachment.go

```diff
@@ -100,6 +104,10 @@ func ListReleaseAttachments(ctx *context.APIContext) {
 	releaseID := ctx.ParamsInt64(":id")
 	release, err := models.GetReleaseByID(ctx, releaseID)
 	if err != nil {
+		if repo_model.IsErrAttachmentNotExist(err) {
```

Comment:
But `GetReleaseID` will never return `ErrAttachmentNotExist`.

Reply:
```suggestion
		if models.IsErrReleaseNotExist(err) {
```

Reply:
Damn... This makes #20891 especially relevant!